### ISSUE TITLE
LINSTOR resizing operation is stuck

### ISSUE THREAD

**Issue by kvaps** [LINBIT/linstor-server#329]
Hello, I just faced with the case when linstor successfully set a new size to the volume definition, but nor drbd device and backing LV weren't resized.

Restart of linstor-sattelite didn't change anything.
Restart of linstor-controller resized the device correctly

linstor 1.19.1
the resource is created in single replica

**Comment by ghernadi** [comment]
ErrorReports or anything suspicious in the logs (controller/satellite)?

**Comment by kvaps** [comment]
Unfortunately logs are clean, after switching logs to TRACE everything works as it should 😕

**Comment by kvaps** [comment]
Today the issue was repeated once again:

```bash
# linstor r l
Defaulted container "linstor-controller" out of: linstor-controller, kube-rbac-proxy
+-------------------------------------------------------------------------------------------------------------------------------------+
| ResourceName                             | Node                   | Port | Usage | Conns |              State | CreatedOn           |
|=====================================================================================================================================|
| pvc-96665a02-7aaa-4f19-b10a-74ec53fac434 | slt-dev-kube-system-01 | 7000 | InUse | Ok    | Resizing, UpToDate | 2022-10-06 09:32:06 |
+-------------------------------------------------------------------------------------------------------------------------------------+
# linstor vd l
Defaulted container "linstor-controller" out of: linstor-controller, kube-rbac-proxy
+------------------------------------------------------------------------------------------------+
| ResourceName                             | VolumeNr | VolumeMinor | Size    | Gross | State    |
|================================================================================================|
| pvc-96665a02-7aaa-4f19-b10a-74ec53fac434 | 0        | 1000        | 100 GiB |       | resizing |
+------------------------------------------------------------------------------------------------+
# linstor vd set-size pvc-96665a02-7aaa-4f19-b10a-74ec53fac434 0 100G
Defaulted container "linstor-controller" out of: linstor-controller, kube-rbac-proxy
SUCCESS:
Description:
    Volume definition with number '0' of resource definition 'pvc-96665a02-7aaa-4f19-b10a-74ec53fac434' modified.
Details:
    Volume definition with number '0' of resource definition 'pvc-96665a02-7aaa-4f19-b10a-74ec53fac434' UUID is: a58b59cd-ce4a-46c2-b9cd-1d7a7eca1b4e
ERROR:
    (Node: 'slt-dev-kube-system-01') Failed to access DRBD super-block of volume pvc-96665a02-7aaa-4f19-b10a-74ec53fac434/0
Show reports:
    linstor error-reports show 639FE3FF-E8C1E-000009
command terminated with exit code 10
# linstor vd l
Defaulted container "linstor-controller" out of: linstor-controller, kube-rbac-proxy
+------------------------------------------------------------------------------------------------+
| ResourceName                             | VolumeNr | VolumeMinor | Size    | Gross | State    |
|================================================================================================|
| pvc-96665a02-7aaa-4f19-b10a-74ec53fac434 | 0        | 1000        | 100 GiB |       | resizing |
+------------------------------------------------------------------------------------------------+
# linstor error-reports show 639FE3FF-E8C1E-000009
ERROR REPORT 639FE3FF-E8C1E-000009

============================================================

Application:                        LINBIT�� LINSTOR
Module:                             Satellite
Version:                            1.20.0
Build ID:                           9c6f7fad48521899f7a99c564b1d33aeacfdbfa8
Build time:                         2022-11-07T16:37:38+00:00
Error time:                         2022-12-28 11:16:25
Node:                               slt-dev-kube-system-01

============================================================

Reported error:
===============

Description:
    Failed to access DRBD super-block of volume pvc-96665a02-7aaa-4f19-b10a-74ec53fac434/0

Category:                           LinStorException
Class name:                         VolumeException
Class canonical name:               com.linbit.linstor.core.devmgr.exceptions.VolumeException
Generated at:                       Method 'hasMetaData', Source file 'DrbdLayer.java', Line #1087

Error message:                      Failed to access DRBD super-block of volume pvc-96665a02-7aaa-4f19-b10a-74ec53fac434/0

Error context:
    An error occurred while processing resource 'Node: 'slt-dev-kube-system-01', Rsc: 'pvc-96665a02-7aaa-4f19-b10a-74ec53fac434''

Call backtrace:

    Method                                   Native Class:Line number
    hasMetaData                              N      com.linbit.linstor.layer.drbd.DrbdLayer:1087
    adjustDrbd                               N      com.linbit.linstor.layer.drbd.DrbdLayer:622
    process                                  N      com.linbit.linstor.layer.drbd.DrbdLayer:396
    process                                  N      com.linbit.linstor.core.devmgr.DeviceHandlerImpl:900
    processResourcesAndSnapshots             N      com.linbit.linstor.core.devmgr.DeviceHandlerImpl:358
    dispatchResources                        N      com.linbit.linstor.core.devmgr.DeviceHandlerImpl:168
    dispatchResources                        N      com.linbit.linstor.core.devmgr.DeviceManagerImpl:309
    phaseDispatchDeviceHandlers              N      com.linbit.linstor.core.devmgr.DeviceManagerImpl:1083
    devMgrLoop                               N      com.linbit.linstor.core.devmgr.DeviceManagerImpl:735
    run                                      N      com.linbit.linstor.core.devmgr.DeviceManagerImpl:631
    run                                      N      java.lang.Thread:829

Caused by:
==========

Category:                           Exception
Class name:                         NoSuchFileException
Class canonical name:               java.nio.file.NoSuchFileException
Generated at:                       Method 'translateToIOException', Source file 'UnixException.java', Line #92

Error message:                      /dev/data/pvc-96665a02-7aaa-4f19-b10a-74ec53fac434_00000

Call backtrace:

    Method                                   Native Class:Line number
    translateToIOException                   N      sun.nio.fs.UnixException:92
    rethrowAsIOException                     N      sun.nio.fs.UnixException:111
    rethrowAsIOException                     N      sun.nio.fs.UnixException:116
    newFileChannel                           N      sun.nio.fs.UnixFileSystemProvider:182
    open                                     N      java.nio.channels.FileChannel:292
    open                                     N      java.nio.channels.FileChannel:345
    readObject                               N      com.linbit.linstor.layer.drbd.utils.MdSuperblockBuffer:74
    hasMetaData                              N      com.linbit.linstor.layer.drbd.DrbdLayer:1082
    adjustDrbd                               N      com.linbit.linstor.layer.drbd.DrbdLayer:622
    process                                  N      com.linbit.linstor.layer.drbd.DrbdLayer:396
    process                                  N      com.linbit.linstor.core.devmgr.DeviceHandlerImpl:900
    processResourcesAndSnapshots             N      com.linbit.linstor.core.devmgr.DeviceHandlerImpl:358
    dispatchResources                        N      com.linbit.linstor.core.devmgr.DeviceHandlerImpl:168
    dispatchResources                        N      com.linbit.linstor.core.devmgr.DeviceManagerImpl:309
    phaseDispatchDeviceHandlers              N      com.linbit.linstor.core.devmgr.DeviceManagerImpl:1083
    devMgrLoop                               N      com.linbit.linstor.core.devmgr.DeviceManagerImpl:735
    run                                      N      com.linbit.linstor.core.devmgr.DeviceManagerImpl:631
    run                                      N      java.lang.Thread:829


END OF ERROR REPORT.
```

The node has no symlink to LV for some reason. It was fixed by recreating symlink manually:

```bash
# lvscan | grep pvc
  ACTIVE            '/dev/data/pvc-96665a02-7aaa-4f19-b10a-74ec53fac434_00000' [100.02 GiB] inherit
# ls -lah /dev/data/pvc-96665a02-7aaa-4f19-b10a-74ec53fac434_00000
ls: cannot access '/dev/data/pvc-96665a02-7aaa-4f19-b10a-74ec53fac434_00000': No such file or directory
# dmsetup ls | grep pvc
data-pvc--96665a02--7aaa--4f19--b10a--74ec53fac434_00000	(253:0)
# ls -lah /dev/dm-* | grep "253, 0"
brw-rw---- 1 root disk 253, 0 Dec 28 10:06 /dev/dm-0
# ln -s /dev/dm-0 /dev/data/pvc-96665a02-7aaa-4f19-b10a-74ec53fac434_00000
# linstor vd set-size pvc-96665a02-7aaa-4f19-b10a-74ec53fac434 0 100G
SUCCESS:
Description:
    Volume definition with number '0' of resource definition 'pvc-96665a02-7aaa-4f19-b10a-74ec53fac434' modified.
Details:
    Volume definition with number '0' of resource definition 'pvc-96665a02-7aaa-4f19-b10a-74ec53fac434' UUID is: a58b59cd-ce4a-46c2-b9cd-1d7a7eca1b4e
```

**Comment by kvaps** [comment]
Guess this is duplicate of https://github.com/LINBIT/linstor-server/issues/326

**Comment by kvaps** [comment]
I was wrong, this isn't duplicate, this is differenet issue

Today I've got the same situation.
Volume definition was resized, but DRBD and backing LV is not:

```bash
# kubectl get pvc -n sentry data-postgres-0
NAME              STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS      AGE
data-postgres-0   Bound    pvc-fa1caf18-b638-4cca-9c2f-e1cdc01b6b3c   56Gi       RWO            linstor-fast-r2   61d

# kubectl get pv pvc-fa1caf18-b638-4cca-9c2f-e1cdc01b6b3c
NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM                    STORAGECLASS      REASON   AGE
pvc-fa1caf18-b638-4cca-9c2f-e1cdc01b6b3c   56Gi       RWO            Delete           Bound    sentry/data-postgres-0   linstor-fast-r2            61d

# linstor vd l -r pvc-fa1caf18-b638-4cca-9c2f-e1cdc01b6b3c
╭────────────────────────────────────────────────────────────────────────────────────────────╮
┊ ResourceName                             ┊ VolumeNr ┊ VolumeMinor ┊ Size   ┊ Gross ┊ State ┊
╞════════════════════════════════════════════════════════════════════════════════════════════╡
┊ pvc-fa1caf18-b638-4cca-9c2f-e1cdc01b6b3c ┊ 0        ┊ 1010        ┊ 56 GiB ┊       ┊ ok    ┊
╰────────────────────────────────────────────────────────────────────────────────────────────╯

# linstor v l -r pvc-fa1caf18-b638-4cca-9c2f-e1cdc01b6b3c
╭────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
┊ Node         ┊ Resource                                 ┊ StoragePool          ┊ VolNr ┊ MinorNr ┊ DeviceName    ┊ Allocated ┊ InUse  ┊      State ┊
╞════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════╡
┊ kube-node-12 ┊ pvc-fa1caf18-b638-4cca-9c2f-e1cdc01b6b3c ┊ DfltDisklessStorPool ┊     0 ┊    1010 ┊ /dev/drbd1010 ┊           ┊ Unused ┊ TieBreaker ┊
┊ kube-node-5  ┊ pvc-fa1caf18-b638-4cca-9c2f-e1cdc01b6b3c ┊ fast                 ┊     0 ┊    1010 ┊ /dev/drbd1010 ┊ 50.01 GiB ┊ Unused ┊   UpToDate ┊
┊ kube-node-7  ┊ pvc-fa1caf18-b638-4cca-9c2f-e1cdc01b6b3c ┊ fast                 ┊     0 ┊    1010 ┊ /dev/drbd1010 ┊ 50.01 GiB ┊ InUse  ┊   UpToDate ┊
╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
```

```
root@kube-node-7:/# lvs | grep pvc-fa1caf18-b638-4cca-9c2f-e1cdc01b6b3c
  pvc-fa1caf18-b638-4cca-9c2f-e1cdc01b6b3c_00000 fast_storage -wi-ao---- 50.01g

root@kube-node-7:/# fdisk -l /dev/drbd1010
Disk /dev/drbd1010: 50 GiB, 53688164352 bytes, 104859696 sectors
Units: sectors of 1 * 512 = 512 bytes
Sector size (logical/physical): 512 bytes / 4096 bytes
I/O size (minimum/optimal): 4096 bytes / 4096 bytes
```

Controller logs has just single error:

```
LINSTOR, Module Controller
Version:            1.20.0 (9c6f7fad48521899f7a99c564b1d33aeacfdbfa8)
Build time:         2022-11-07T16:37:38+00:00
Java Version:       11
Java VM:            Ubuntu, Version 11.0.16+8-post-Ubuntu-0ubuntu122.04
Operating system:   Linux, Version 5.3.0-51-generic
Environment:        amd64, 1 processors, 7737 MiB memory reserved for allocations


System components initialization in progress

Loading configuration file "/etc/linstor/linstor.toml"
16:53:16.518 [main] INFO  LINSTOR/Controller - SYSTEM - ErrorReporter DB first time init.
16:53:16.519 [main] INFO  LINSTOR/Controller - SYSTEM - Log directory set to: '/var/log/linstor-controller'
16:53:16.547 [main] INFO  LINSTOR/Controller - SYSTEM - Database type is Kubernetes-CRD
16:53:16.547 [Main] INFO  LINSTOR/Controller - SYSTEM - Loading API classes started.
16:53:16.829 [Main] INFO  LINSTOR/Controller - SYSTEM - API classes loading finished: 282ms
16:53:16.829 [Main] INFO  LINSTOR/Controller - SYSTEM - Dependency injection started.
16:53:16.841 [Main] INFO  LINSTOR/Controller - SYSTEM - Attempting dynamic load of extension module "com.linbit.linstor.modularcrypto.FipsCryptoModule"
16:53:16.841 [Main] INFO  LINSTOR/Controller - SYSTEM - Extension module "com.linbit.linstor.modularcrypto.FipsCryptoModule" is not installed
16:53:16.841 [Main] INFO  LINSTOR/Controller - SYSTEM - Attempting dynamic load of extension module "com.linbit.linstor.modularcrypto.JclCryptoModule"
16:53:16.846 [Main] INFO  LINSTOR/Controller - SYSTEM - Dynamic load of extension module "com.linbit.linstor.modularcrypto.JclCryptoModule" was successful
16:53:16.846 [Main] INFO  LINSTOR/Controller - SYSTEM - Attempting dynamic load of extension module "com.linbit.linstor.spacetracking.ControllerSpaceTrackingModule"
16:53:16.847 [Main] INFO  LINSTOR/Controller - SYSTEM - Extension module "com.linbit.linstor.spacetracking.ControllerSpaceTrackingModule" is not installed
16:53:17.436 [Main] INFO  LINSTOR/Controller - SYSTEM - Dependency injection finished: 607ms
16:53:17.436 [Main] INFO  LINSTOR/Controller - SYSTEM - Cryptography provider: Using default cryptography module
16:53:17.608 [Main] INFO  LINSTOR/Controller - SYSTEM - Initializing authentication subsystem
16:53:17.826 [Main] INFO  LINSTOR/Controller - SYSTEM - Starting service instance 'TimerEventService' of type TimerEventService
16:53:17.826 [Main] INFO  LINSTOR/Controller - SYSTEM - Initializing the k8s crd database connector
16:53:17.826 [Main] INFO  LINSTOR/Controller - SYSTEM - Kubernetes-CRD connection URL is "k8s"
16:53:18.738 [Main] INFO  LINSTOR/Controller - SYSTEM - Starting service instance 'K8sCrdDatabaseService' of type K8sCrdDatabaseService
16:53:18.744 [Main] INFO  LINSTOR/Controller - SYSTEM - Loading security objects
16:53:18.830 [Main] INFO  LINSTOR/Controller - SYSTEM - Current security level is NO_SECURITY
16:53:19.103 [Main] INFO  LINSTOR/Controller - SYSTEM - Core objects load from database is in progress
16:53:19.501 [Main] INFO  LINSTOR/Controller - SYSTEM - Core objects load from database completed
16:53:19.799 [Main] INFO  LINSTOR/Controller - SYSTEM - Starting service instance 'TaskScheduleService' of type TaskScheduleService
16:53:19.801 [Main] INFO  LINSTOR/Controller - SYSTEM - Initializing network communications services
16:53:19.987 [Main] WARN  LINSTOR/Controller - SYSTEM - NetComService: Connector DebugSslConnector: Binding the socket to the IPv6 anylocal address failed, attempting fallback to IPv4
16:53:19.987 [Main] INFO  LINSTOR/Controller - SYSTEM - Created network communication service 'DebugSslConnector', bound to ::0:3373
16:53:19.988 [Main] INFO  LINSTOR/Controller - SYSTEM - Created network communication service 'PlainConnector'
16:53:20.098 [Main] INFO  LINSTOR/Controller - SYSTEM - Created network communication service 'SslConnector'
16:53:20.098 [Main] INFO  LINSTOR/Controller - SYSTEM - Reconnecting to previously known nodes
16:53:20.135 [Main] INFO  LINSTOR/Controller - SYSTEM - Reconnect requests sent
16:53:20.136 [Main] INFO  LINSTOR/Controller - SYSTEM - Starting service instance 'ScheduleBackupService' of type ScheduleBackupService
16:53:20.137 [Main] INFO  LINSTOR/Controller - SYSTEM - Starting service instance 'EbsStatusPoll' of type EbsStatusPoll
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.sun.xml.bind.v2.runtime.reflect.opt.Injector (file:/usr/share/linstor-server/lib/jaxb-impl-2.2.11.jar) to method java.lang.ClassLoader.defineClass(java.lang.String,byte[],int,int)
WARNING: Please consider reporting this to the maintainers of com.sun.xml.bind.v2.runtime.reflect.opt.Injector
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
Dec 28, 2022 4:53:20 PM org.glassfish.jersey.internal.Errors logErrors
WARNING: The following warnings have been detected: WARNING: A resource, Resource{"v1/action", 0 child resources, 0 resource methods, 0 sub-resource locator, 0 method handler classes, 0 method handler instances}, with path "v1/action" is empty. It has no resource (or sub resource) methods neither sub resource locators defined.

Dec 28, 2022 4:53:20 PM org.glassfish.jersey.internal.Errors logErrors
WARNING: The following warnings have been detected: WARNING: A resource, Resource{"v1/action", 0 child resources, 0 resource methods, 0 sub-resource locator, 0 method handler classes, 0 method handler instances}, with path "v1/action" is empty. It has no resource (or sub resource) methods neither sub resource locators defined.

Dec 28, 2022 4:53:21 PM org.glassfish.grizzly.http.server.NetworkListener start
INFO: Started listener bound to [127.0.0.1:3370]
Dec 28, 2022 4:53:21 PM org.glassfish.grizzly.http.server.HttpServer start
INFO: [HttpServer] Started.
Dec 28, 2022 4:53:21 PM org.glassfish.grizzly.http.server.NetworkListener start
INFO: Started listener bound to [0.0.0.0:3371]
Dec 28, 2022 4:53:21 PM org.glassfish.grizzly.http.server.HttpServer start
INFO: [HttpServer-1] Started.
16:53:21.016 [Main] WARN  LINSTOR/Controller - SYSTEM - Not calling 'systemd-notify' as NOTIFY_SOCKET is null
16:53:21.016 [Main] INFO  LINSTOR/Controller - SYSTEM - Controller initialized

16:53:21.068 [TaskScheduleService] INFO  LINSTOR/Controller - SYSTEM - LogArchive: Running log archive on directory: /var/log/linstor-controller
16:53:21.069 [TaskScheduleService] INFO  LINSTOR/Controller - SYSTEM - LogArchive: No logs to archive.
16:50:55.843 [SslConnector] ERROR LINSTOR/Controller - SYSTEM - Target decrypted buffer is too small! [Report number 63AC747C-00000-000000]

16:53:19.802 [TaskScheduleService] INFO  LINSTOR/Controller - SYSTEM - LogArchive: Running log archive on directory: /var/log/linstor-controller
16:53:19.804 [TaskScheduleService] INFO  LINSTOR/Controller - SYSTEM - LogArchive: No logs to archive.
16:53:19.803 [TaskScheduleService] INFO  LINSTOR/Controller - SYSTEM - LogArchive: Running log archive on directory: /var/log/linstor-controller
16:53:19.804 [TaskScheduleService] INFO  LINSTOR/Controller - SYSTEM - LogArchive: No logs to archive.
16:53:19.801 [TaskScheduleService] INFO  LINSTOR/Controller - SYSTEM - LogArchive: Running log archive on directory: /var/log/linstor-controller
16:53:19.804 [TaskScheduleService] INFO  LINSTOR/Controller - SYSTEM - LogArchive: No logs to archive.
16:53:19.802 [TaskScheduleService] INFO  LINSTOR/Controller - SYSTEM - LogArchive: Running log archive on directory: /var/log/linstor-controller
16:53:19.804 [TaskScheduleService] INFO  LINSTOR/Controller - SYSTEM - LogArchive: No logs to archive.
16:53:19.802 [TaskScheduleService] INFO  LINSTOR/Controller - SYSTEM - LogArchive: Running log archive on directory: /var/log/linstor-controller
16:53:19.804 [TaskScheduleService] INFO  LINSTOR/Controller - SYSTEM - LogArchive: No logs to archive.
16:53:19.804 [TaskScheduleService] INFO  LINSTOR/Controller - SYSTEM - LogArchive: Running log archive on directory: /var/log/linstor-controller
16:53:19.806 [TaskScheduleService] INFO  LINSTOR/Controller - SYSTEM - LogArchive: No logs to archive.
[custis] root@kube-master-1 ~ #
```


```
ERROR REPORT 63AC747C-00000-000000

============================================================

Application:                        LINBIT�� LINSTOR
Module:                             Controller
Version:                            1.20.0
Build ID:                           9c6f7fad48521899f7a99c564b1d33aeacfdbfa8
Build time:                         2022-11-07T16:37:38+00:00
Error time:                         2022-12-29 16:50:55
Node:                               linstor-controller-857684db56-dvs9d

============================================================

Reported error:
===============

Category:                           Error
Class name:                         ImplementationError
Class canonical name:               com.linbit.ImplementationError
Generated at:                       Method 'unwrapInto', Source file 'SslTcpConnectorPeer.java', Line #241

Error message:                      Target decrypted buffer is too small!

Call backtrace:

    Method                                   Native Class:Line number
    unwrapInto                               N      com.linbit.linstor.netcom.ssl.SslTcpConnectorPeer:241
    readData                                 N      com.linbit.linstor.netcom.ssl.SslTcpConnectorPeer:287
    read                                     N      com.linbit.linstor.netcom.ssl.SslTcpConnectorPeer:195
    run                                      N      com.linbit.linstor.netcom.TcpConnectorService:543
    run                                      N      java.lang.Thread:829


END OF ERROR REPORT.
```

Satellite logs are clean.

Restart of linstor-controller resized LV, but not DRBD:

```
root@kube-node-7:/# lvs | grep pvc-fa1caf18-b638-4cca-9c2f-e1cdc01b6b3c
  pvc-fa1caf18-b638-4cca-9c2f-e1cdc01b6b3c_00000 fast_storage -wi-ao---- <56.02g    

root@kube-node-7:/# fdisk -l /dev/drbd1010
Disk /dev/drbd1010: 50 GiB, 53688164352 bytes, 104859696 sectors
Units: sectors of 1 * 512 = 512 bytes
Sector size (logical/physical): 512 bytes / 4096 bytes
I/O size (minimum/optimal): 4096 bytes / 4096 bytes
```

Restart of linstor-satellite didn't change anything. I was forced to execute resize manually:

```
# drbdadm resize pvc-fa1caf18-b638-4cca-9c2f-e1cdc01b6b3c
```

only after that it become correct size:
```
# fdisk -l /dev/drbd1010
Disk /dev/drbd1010: 56 GiB, 60133433344 bytes, 117448112 sectors
Units: sectors of 1 * 512 = 512 bytes
Sector size (logical/physical): 512 bytes / 4096 bytes
I/O size (minimum/optimal): 4096 bytes / 4096 bytes
```

**Comment by duckhawk** [comment]
I have exact the same problem. And after manual resize force I still have problem on diskless node:
```
root@node3:/# linstor v l | grep 557
| node3   | pvc-55789aab-5c34-4df7-b270-f58647822c54 | DfltDisklessStorPool |     0 |    1074 | /dev/drbd1074 |            | InUse  | Diskless |
| node4   | pvc-55789aab-5c34-4df7-b270-f58647822c54 | store                |     0 |    1074 | /dev/drbd1074 | 930.15 MiB | Unused | UpToDate |
root@node3:/# linstor vd l | grep 557
| pvc-55789aab-5c34-4df7-b270-f58647822c54 | 0        | 1074        | 10 GiB  |       | ok    |
root@node3:/# lsblk /dev/drbd1074
NAME     MAJ:MIN  RM SIZE RO TYPE MOUNTPOINTS
drbd1074 147:1074  0   1G  0 disk 
```
